app/pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   carousel_slider: ^5.0.0
   flutter:
     sdk: flutter
+  provider: ^6.1.2
```

Comment:
Is it ok to bring provider into the mix for demonstration purpose? (having a working increment/decrement in widgetbook)